tests/unit/scatter-test.js

```diff
@@ -105,3 +106,18 @@ test('Test domain without domain padding', function(assert) {
   assert.equal(component.get('xScale')(0.017), 497.65);
   assert.equal(component.get('yScale')(0.03), 193.815);
 });
+
+test('Test legend color override', function(assert) {
+  var component = this.subject(scatterContent);
```

Comment:
> Or was that part of the test added after your comment?
 
yes, looks like it, according to the "outdated diff" on the similar comment you made later:
https://github.com/Addepar/ember-charts/pull/152/files/cb4f443a829ef459b341431c2e0dd4489ca86177#r54045960